components/github/README.md

```diff
@@ -1,257 +1,12 @@
-# Github API Integration Platform
+# Getting Started
 
-### Connect Github + 1000s of apps, remarkably fast.
+## Github Triggers: Webhooks vs. Polling
+The Github triggers are built in way that configures the trigger as a webhook if you are the admin of a repo, and falls back to polling if you are not the admin of a repo.
 
----
+**Example: New or Updated Issue (Admin)**
```

Comment:
What does (Admin) mean here? That's a little confusing to me

Reply:
Dropped this part, and just explained with the two examples.